ddpui/tests/services/test_org_cleanup_service.py

```
@@ -0,0 +1,237 @@
+
+    # Setup OrgDataFlowv1 and DataflowOrgTask (manual dataflow with airbyte tasks)
+    dataflow_manual = OrgDataFlowv1.objects.create(
+        org=org, dataflow_type="manual", deployment_id="deploy1", deployment_name="df1"
+    )
+    DataflowOrgTask.objects.create(dataflow=dataflow_manual, orgtask=orgtask_sync)
+
+    # Setup orchestrate pipeline with both sync and dbt tasks
+    dataflow_orchestrate = OrgDataFlowv1.objects.create(
+        org=org, dataflow_type="orchestrate", deployment_id="deploy2", deployment_name="df2"
+    )
+    dfo_sync = DataflowOrgTask.objects.create(dataflow=dataflow_orchestrate, orgtask=orgtask_sync)
```

Ruff (0.14.10)

Local variable `dfo_sync` is assigned to but never used

Remove assignment to unused variable `dfo_sync`

(F841)